### PR TITLE
UINV-128 Accessibility error: ID attribute value must be unique

### PR DESCRIPTION
**Pull request by AndreiShumski2**
## Purpose
https://issues.folio.org/browse/UINV-128

## Pre-Merge Checklist
Before merging this PR, please go through the following list and take appropriate actions.
- [x] I've added appropriate record to the CHANGELOG.md
- Does this PR meet or exceed the expected quality standards?
  - [x] Code coverage on new code is 80% or greater
  - [x] Duplications on new code is 3% or less
  - [x] There are no major code smells or security issues
- Does this introduce breaking changes?
  - [x] If any API-related changes - okapi interfaces and permissions are reviewed/changed correspondingly
  - [x] There are no breaking changes in this PR.

If there are breaking changes, please **STOP** and consider the following:

- What other modules will these changes impact?
- Do JIRAs exist to update the impacted modules?
  - [ ] If not, please create them
  - [ ] Do they contain the appropriate level of detail?  Which endpoints/schemas changed, etc.
  - [ ] Do they have all they appropriate links to blocked/related issues?
- Are the JIRAs under active development?
  - [ ] If not, contact the project's PO and make sure they're aware of the urgency.
- Do PRs exist for these changes?
  - [ ] If so, have they been approved?

Ideally all of the PRs involved in breaking changes would be merged in the same day to avoid breaking the folio-testing environment.  Communication is paramount if that is to be achieved, especially as the number of intermodule and inter-team dependencies increase.

While it's helpful for reviewers to help identify potential problems, ensuring that it's safe to merge is ultimately the responsibility of the PR assignee.

### CODE REVIEW

File: src/invoices/InvoiceForm/InvoiceForm.js [hunk 1]
```diff
@@ -55,6 +55,7 @@ import {
 import {
   ApprovedBy,
 } from '../../common/components';
+import { SECTIONS_INVOICE as SECTIONS } from '../constants';
```
Comment (aliaksei-chumakou) [inline src/invoices/InvoiceForm/InvoiceForm.js:58]: I don't see that `SECTIONS_INVOICE` is a new object. There are section ids for Invoice details, they might differ from the Invoice form, that's why we had `SECTIONS` here

File: src/invoices/InvoiceForm/InvoiceForm.js [hunk 2]
```diff
@@ -435,7 +428,7 @@ class InvoiceForm extends Component {
                     </Row>
                   </Accordion> */}
                   <Accordion
-                    id={SECTIONS.documents}
+                    id={SECTIONS.DOCUMENTS}
```
Comment (aliaksei-chumakou) [inline src/invoices/InvoiceForm/InvoiceForm.js:431]: `SECTIONS.documents` was looking better, `SECTIONS` is a constant, `documents` is just a key

File: src/invoices/InvoiceForm/InvoiceForm.js (hunk 1 above)
Comment (AndreiShumski2) [reply src/invoices/InvoiceForm/InvoiceForm.js:58]: Invoice details have the same sections as an Invoice form, except 'Extended Information'. I don't think we need to keep the same const in two places.

File: src/invoices/InvoiceForm/InvoiceForm.js (hunk 2 above)
Comment (AndreiShumski2) [reply src/invoices/InvoiceForm/InvoiceForm.js:431]: I'll switch keys to lowerCase

File: CHANGELOG.md [hunk 3]
```diff
@@ -7,6 +7,8 @@
 * [UINV-106](https://issues.folio.org/browse/UINV-106) Display Batch vouchers in settings
 
 ### Bug fixes
+
```
Comment (AndreiShumski2) [inline CHANGELOG.md:10]: -

**Pull request merged**